fdbserver/DataDistributionTracker.actor.cpp

```diff
@@ -189,6 +201,11 @@ ACTOR Future<Void> trackShardBytes(
 
 			Transaction tr(self->cx);
 			StorageMetrics metrics = wait( tr.waitStorageMetrics( keys, bounds.min, bounds.max, bounds.permittedError, CLIENT_KNOBS->STORAGE_METRICS_SHARD_LIMIT ) );
+			BandwidthStatus newBandwidthStatus = getBandwidthStatus( metrics );
+			if(newBandwidthStatus == BandwidthStatusLow && bandwidthStatus != BandwidthStatusLow) {
+				lastLowBandwidthTime = now();
```

Comment:
I think the name of this might cause some confusion, as it to me implies the last time the shard was at low bandwidth rather than the time when low bandwidth started. Maybe something like `lastLowBandwidthStartTime` is clearer?